components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -82,10 +82,7 @@ impl Manifest {
             None => pkg_ident.origin + "/" + &pkg_ident.name,
         };
 
-        let binds: Vec<String> = match matches.values_of("BIND") {
-            Some(binds) => binds.map(|s| s.to_string()).collect(),
-            None => Vec::new(),
-        };
+        let binds = bind::parse_bind_args(&matches)?;
```

Comment:
I'm not sold on the utility of pushing CLAP awareness further down into the `Bind` type. Is there something additional you're shooting for by doing that?  Seems like using the parse method in here with the prior implementation would be a good separation of concerns.